integration-cli/docker_cli_build_test.go

```diff
@@ -4583,17 +4583,21 @@ func TestBuildRenamedDockerfile(t *testing.T) {
 		t.Fatalf("Should have used dFile, output:%s", out)
 	}
 
-	out, _, err = dockerCmdInDir(t, ctx.Dir, "build", "--file=/etc/passwd", "-t", "test5", ".")
+	dirWithNoDockerfile, err := ioutil.TempDir(os.TempDir(), "empty")
+	if err != nil {
+		t.Fatal(err)
+	}
+	out, _, err = dockerCmdInDir(t, ctx.Dir, "build", fmt.Sprintf("--file=%s", dirWithNoDockerfile), "-t", "test5", ".")
```

Comment:
@duglin ah, makes sense. For some reason I guess I thought /etc/passwd as a dir. :-/ I'll fix this.